src/App.vue

```diff
@@ -2,7 +2,7 @@
   <div id="app" class="min-h-screen">
     <div class="content min-h-screen max-w-desktop">
       <div class="min-h-screen wrapper" ref="wrapper">
-        <router-view></router-view>
+        <router-view :showLoading="showLoading"></router-view>
```

Comment:
I think we should't give props to router-view

Reply:
@kenodressel what do you think=

Reply:
I think it should be solved differently. You could, for example, only render the router-view if the loading is done, if the data is so vital to the application. Otherwise I would recommend handling the loading state in the individual components which are depending on the data.